capstone/capweb/templates/includes/nav.html

```diff
@@ -35,6 +35,7 @@
           <a class="dropdown-item" href="{% url "tools" %}">Overview</a>
           <a class="dropdown-item" href="{% url "api" %}">API</a>
           <a class="dropdown-item" href="{% url "bulk-docs" %}">Bulk Data</a>
+          <a class="dropdown-item" href="{% url "search" %}">Search</a>
```

Comment:
Let's not include this endpoint in the nav yet, so we can kick the tires on prod before sharing it.

Reply:
Ok

Reply:
Commented it out in the template